### PR TITLE
drivers: replace Sony licenses with Apache 2.0

### PR DESCRIPTION
**Pull request by jerpelea**
## Summary
drivers: replace Sony licenses with Apache 2.0

## Impact
None

## Testing
None

### CODE REVIEW

**Review by xiaoxiang781216 — COMMENTED** [review]
LGTM, once the nxstyle warning get clear, I will merge it.

**Review by davids5 — APPROVED** [review]
LGTM And NXStyle changes!

**Pull request merged**
